myWidget/Views/myWidgetEntryView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  myWidgetEntryView.swift
+//  포스밀
+//
+//  Created by Kim Insub on 2022/07/23.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct myWidgetEntryView : View {
+
+    //Define
+    @Environment(\.widgetFamily) var family
+
+    var entry: Provider.Entry
```

Comment:
```suggestion
    private var entry: Provider.Entry
```

private 설정이 가능해보이네요 ~

Reply:
private이라면 init할때에 param으로 인식하지 못합니다